artifact/checksum.go

```diff
@@ -16,9 +16,9 @@ package artifact
 
 import (
 	"bytes"
-	"github.com/minio/sha256-simd"
 	"encoding/hex"
 	"fmt"
+	"github.com/minio/sha256-simd"
```

Comment:
Minor comment: can you please move this line to the second group of imports to group github.com items together?

Reply:
Done!